extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsRuntimeConfig.java

```diff
@@ -39,10 +40,18 @@
 
     /**
      * A comma-separated list of topic names.
-     * The pipeline will only be started once all these topics are present in the Kafka cluster.
+     * The pipeline will only be started once all these topics are present in the Kafka cluster
+     * and {@code ignore.topics} is set to false.
      */
     @ConfigItem
-    public List<String> topics;
+    public Optional<List<String>> topics;
+
+    /**
+     * Timeout to wait for topic names to be returned from admin client.
+     * If set to 0 (or negative), {@code topics} check is ignored.
+     */
+    @ConfigItem(defaultValue = "10S")
```

Comment:
```suggestion
    @ConfigItem(defaultValue = "10")
```
For seconds, we can even omit the unit and the duration converter will be considered as seconds.

Reply:
Yes but IMHO it's better for documentation to have the unit.

Reply:
Agree. But If we are talking about the doc generator, it is already aware of it and it generates default value with a proper unit even when the unit is omitted.